File: Mixter/Domain/Subscription.cs
namespace Mixter.Domain
{
    public class Subscription
    {
        public static void FollowUser(IEventPublisher eventPublisher, UserId follower, UserId followee)
        {
            eventPublisher.Publish(new UserFollowed(new SubscriptionId(follower, followee)));
        }
    }
}

File: app/Domain/Subscriptions/Subscription.php
<?php

namespace App\Domain\Subscriptions;

use App\Domain\Identity\UserId;
use App\Domain\IEventPublisher;

class Subscription
{
    public static function followUser(IEventPublisher $eventPublisher, UserId $followerId, UserId $followeeId): void
    {
        $eventPublisher->publish(
            new UserFollowed(
                new SubscriptionId($followerId,$followeeId)
            )
        );
    }
}
